cw-rct201/src/App.tsx

```diff
@@ -1,34 +0,0 @@
-import { Button } from "@chakra-ui/react";
-import React from "react";
-import { ToastContainer,toast } from "react-toastify";
-import "react-toastify/dist/ReactToastify.css"
-import "./App.css";
-
-
-
-import Footer from "./Components/Footer/Footer";
-import Navbar from "./Components/Navbar/Navbar";
-
-import AllRoutes from "./Routes/AllRoutes";
-
-function App() {
-  return (
-    <div className="App">
-
-   
-      <Navbar />
-      <AllRoutes />
-      <ToastContainer/>
-       <Footer />
```

Comment:
Fix indentation for this file

Reply:
Fixed Indentation sir and now its  formatted by prettier also